### ISSUE TITLE
Import/Export: Content-Type is reset

### ISSUE THREAD

**Issue by hakoerber** [Waboodoo/HTTP-Shortcuts#113]
Hello,

I have an issue with the import/export feature. 

## Steps to reproduce:

1. When configuring a POST request, I have set `Content-Type` of the Request Body to `application/json` in the app. Everything works fine.
2. I export to local storage as JSON
3. I import the exact same file (no modification)
4. The `Content-Type` field (in the app) is (re)set to `text/plain`. The call now fails obviously.

When I look into the exported JSON file, I see that Content-Type is set correctly:

```json
{
  "categories": [
    {
      "id": 1,
      "layoutType": "linear_list",
      "name": "[...]",
      "shortcuts": [
        {
          "[...]"
          "contentType": "application/json",
          "[...]"
        },
      ]
    }
  ]
"[...]"
}
```

## More info

App version is `1.23.0`

---

It looks like the import does not import the `contentType` field correctly.
Is there anything else I can provide to help troubleshooting?

**Comment by Waboodoo** [comment]
Thanks for reporting. I will look into it and try to fix it for the next release.

**Comment by Waboodoo** [comment]
This is fixed in version 1.24, which is expected to be released in about 2 weeks.
